lib/rule_engine/ast.py

```diff
@@ -237,6 +238,19 @@ def from_string(cls, context, string):
 			dt = dt.replace(tzinfo=context.default_timezone)
 		return cls(context, dt)
 
+class TimedeltaExpression(LiteralExpressionBase):
+	"""
+	Literal timedelta expressions representing an offset from a specific point in time. This expression type always evaluates to true.
```

Comment:
```suggestion
	Literal timedelta expressions representing an offset from a specific point in time.
```

Removes the second sentence as it appears to be inaccurate:
```
rule > t"P0D" ? true : false
result: 
False
rule > t"P1D" ? true : false
result: 
True
rule >